package.json

```diff
@@ -19,12 +18,9 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "start:prod": "cross-env REACT_APP_QUERY_NODE_URL=https://hydra.joystream.org/graphql REACT_APP_ORION_URL=https://orion.joystream.org/graphql REACT_APP_STORAGE_NODE_URL=https://rome-rpc-endpoint.joystream.org/asset/v0/ yarn start",
     "dev": "yarn start",
     "dev:prod": "yarn start:prod",
```

Comment:
It should be deleted as well.

Reply:
Certainly should be, my fault during rebase, removed now